admin_manual/groupware/mail.rst

```diff
@@ -127,3 +127,10 @@ multiple days.
 Each sent email will be analyzed by an AI to check whether a reply is expected.
 
 The feature can be enabled through the global :ref:`mail_llm_processing` setting.
+
+Translation
+-----------
+
+.. versionadded:: 4.2
+
+The mail app can optionally provide translations for messages if the `translation API <https://docs.nextcloud.com/server/latest/admin_manual/ai/app_assistant.html#machine-translation>`_ is enabled.
```

Comment:
Apologies for not being clearer but please don't use an absolute link. Use a reference for the hyperlink. We not only generate an online documentation, but also a PDF. The PDF would otherwise open the external link, when it could link inside of the document.
The second problem of the absolute link is also that it will always go to the *latest* version. At some point the current master will become stable30. Then you open the docs for 30 and end up at 31 or even newer.
Lastly, with an anchor and internal reference, sphinx will actually ensure that you are linking to something that exists. We don't run into the problem of dangling/dead links.

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html

Reply:
Thanks a lot for the detailed explanation <3